### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by lupa18** [xmikos/fdroiddata#22]
I'm not sure if it the correct place to open this issue, my apologies if not. 

I generate a new user in my CyanogenMod 12.1 and download and install f-droid and then tried to install libresignal but failed: "Insallation fail due an unknown error". 

thanks in advance.

**Comment by mimi89999** [comment]
Have you uninstalled Signal before?

**Comment by mimi89999** [comment]
https://github.com/LibreSignal/LibreSignal/issues/25

**Comment by lupa18** [comment]
No. But I'm using another user space. There I have installed f-droid from scratch and tried to install libresignal without success.

**Comment by mimi89999** [comment]
Could you please post a logcat?

**Comment by lupa18** [comment]
Is this enough? https://ptpb.pw/AH7Zcsaka8JdPRkcqGmdccudVDrx.txt
